### PR TITLE
Update getting started guide

### PR DESCRIPTION
**Pull request by cdrage**
Updates the getting started guide with a (slightly) improved format.

### CODE REVIEW

**Comment by surajnarwade** [comment]
LGTM

File: docs/getting-started.md [hunk 1]
```diff
@@ -69,35 +62,48 @@ The server is accessible via web console at:
...
+Resolving deltas: 100% (224/224), done.
+
+$ cd nodejs-ex 
+~/nodejs-ex  master
+```
+
+Now that you've created an application, add a component of type _nodejs_ to the application, from the current directory where our code lies:
 
 ```console
 $ odo create nodejs --local=.
```
Comment (kadel) [inline docs/getting-started.md]: the output shown here is from really old version.
It looks completely different now.

File: docs/getting-started.md [hunk 2]
```diff
@@ -120,12 +126,13 @@ Pushing image 172.30.1.1:5000/myproject/nodejs:latest ...
 Push successful
 ```
 
-Great news! Your component has been deployed on OpenShift now. Let's quickly check how it looks!
+Great news! Your component has been deployed to OpenShift! Now we'll connect to the component.
```
Comment (kadel) [inline docs/getting-started.md]: nope ;-)
There is one missing step. You have to run `odo push` as `odo create --local` doesn't do push anymore.

File: docs/getting-started.md [hunk 3]
```diff
@@ -120,12 +126,13 @@ Pushing image 172.30.1.1:5000/myproject/nodejs:latest ...
...
 
-##### Connect to the component
+### 4. Accessing the component
 
-We need create the URL so we can connect to it our application.
-```
+To access the component, we'll need to create an OpenShift endpoint:
```
Comment (kadel) [inline docs/getting-started.md]: In OpenShift and Kubernetes endpoint means something completely different. We should use `endpoint` in this context.

File: docs/getting-started.md [hunk 4]
```diff
@@ -135,30 +142,33 @@ nodejs - nodejs-myproject.192.168.42.147.nip.io
...
 
-Well, your application looks great, but now you've made some changes in the code. Let's deploy these changes and see how it looks.
+The current component is already set to nodejs, which we can confirm from using: `odo component get`.
 
-The current component is already set to nodejs, which can confirm from `odo component get`, so all we need to do is -
+Now let's push the changes:
 
 ```console
 $ odo push
 pushing changes to component: nodejs
```
Comment (kadel) [inline docs/getting-started.md]: again, the output is different that it was before

File: docs/getting-started.md (hunk 3 above)
Comment (cdrage) [reply docs/getting-started.md]: you mean `route` ?

**Comment by cdrage** [comment]
@kadel Thanks for being patient. I've update the PR again since I missed the other section with the updated outpush (`odo push`).

Ready for another review!

**Pull request merged**